### ISSUE TITLE
Documentation colors & WCAG

### ISSUE THREAD

**Issue by Jeger** [htmlburger/carbon-fields#76]
Hi, first of all. Thank you so much for creating the custom field plugin that the Wordpress community needed! I is a joy to use.

One small request, on your website you have very small, light gray text.
This is not very readable, and I'm guessing if I think it's hard to read someone else might find it impossible.

I checked both the body text, and the code examples with green on gray. And both fail WCAG 2.0 tests.
IMO you should aim to pass WCAG 2.0 AAA tests for contrast on text like documentation.
As it not only makes it accessible to everyone, but more usable for everyone if you spend a couple of hours reading it :)

Thanks again.

**Comment by emohamed** [comment]
Hi @Jeger!

Thank you for bringing this to our attention! We'll work with our design guy to fix the issue.

**Comment by emohamed** [comment]
We've applied some changes that improve to the readability of the docs pages. 

Let me know if you notice other problems with the usability of the site.
